### PR TITLE
kind of affiliated, actually

### PR DESCRIPTION
**Pull request by isaacs**
Correct the readme which says I'm not affiliated with GitHub, which hasn't been true for a few months now. 😬

@clarkbw I see you're on this repo/ Can I get a +1 on this from a GitHub product pov?  Or do you know who else might have corporate feels about the wording?  I'm not precious about it, just trying to be precise.

### CODE REVIEW

**Review by erkinalp — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,8 +2,16 @@
...
+<del>I'm not affiliated with GitHub in any way, except that I use it all
+day long, and almost all my code is hosted there.</del>
+
+<ins>UPDATE: as of April 2020, with [the acquisition of npm, Inc. by
+GitHub](https://blog.npmjs.org/post/612764866888007680/next-phase-montage),
+I am a GitHub employee.  I remain focused on the
+[npm/cli](https://github.com/npm/cli) project, and any opinions expressed
+here belong to the person expressing them at the time they were expressed,
+and should not be taken as official GitHub policy in any way.  I still use
+GitHub all day long, and almost all my code is still hosted there.</ins>
```
Comment (clarkbw) [inline README.md:14]: > and _almost_ all my code is still hosted there

We still haven't reached 100%?  😸

**Comment by clarkbw** [comment]
> Or do you know who else might have corporate feels about the wording? I'm not precious about it, just trying to be precise.

I'm happy to sign off here and keep the corporate people at bay 😉

File: README.md (hunk 1 above)
Comment (erkinalp) [reply README.md:14]: @clarkbw He will reach 100% when GitHub becomes Microsoft Git Hub :clap:

File: README.md [hunk 2]
```diff
@@ -2,8 +2,16 @@
 
 **This is not the actual repository for the GitHub website.**
 
-I'm not affiliated with GitHub in any way, except that I use it all
-day long, and almost all my code is hosted there.
+<del>I'm not affiliated with GitHub in any way, except that I use it all
+day long, and almost all my code is hosted there.</del>
+
+<ins>UPDATE: as of April 2020, with [the acquisition of npm, Inc. by
```
Comment (safinsingh) [inline README.md:8]: typo ([the should be the)

**Review by safinsingh — CHANGES_REQUESTED** [review]
See comments

File: README.md (hunk 1 above)
Comment (clarkbw) [reply README.md:14]: > @clarkbw He will reach 100% when GitHub becomes Microsoft Git Hub 👏

Microsoft GitHub Professional Edition 365 😉

File: README.md (hunk 2 above)
Comment (dorianmarie) [reply README.md:8]: nope, it's part of a link

**Review by ruyadorno — APPROVED** [review]

**Pull request merged**